modules/default.nix

```diff
@@ -14,7 +14,5 @@
       "Additional workarounds are no longer required for Docker to work. Please use the standard `virtualisation.docker` NixOS options.")
     (lib.mkRemovedOptionModule [ "wsl" "interop" "preserveArgvZero" ]
       "wsl.interop.preserveArgvZero is now always enabled, as used by modern WSL versions.")
-    (lib.mkRemovedOptionModule [ "wsl" "tarball" ]
```

Comment:
This needs to be adjusted, not removed, because there were other options in this namespace.